server/routes/appointments/create-and-edit/handlers/comment.ts

```diff
@@ -1,12 +1,14 @@
 import { Request, Response } from 'express'
 import { Expose } from 'class-transformer'
+import { MaxLength } from 'class-validator'
 import EditAppointmentService from '../../../../services/editAppointmentService'
 import { AppointmentJourneyMode } from '../appointmentJourney'
 import { YesNo } from '../../../../@types/activities'
 import { getAppointmentBackLinkHref, isApplyToQuestionRequired } from '../../../../utils/editAppointmentUtils'
 
 export class Comment {
   @Expose()
+  @MaxLength(4000, { message: 'You must enter a comment which has no more than 4,000 characters' })
```

Comment:
Any chance of a test for this? There's a comparable one you can take here https://github.com/ministryofjustice/hmpps-activities-management/blob/main/server/routes/appointments/create-and-edit/handlers/bulk-appointments/bulkAppointmentAddComment.test.ts#L163

Reply:
Added the test